test/functional/feature_reindex.py

```diff
@@ -22,7 +22,7 @@ def reindex(self, justchainstate=False):
         self.generatetoaddress(self.nodes[0], 3, self.nodes[0].get_deterministic_priv_key().address)
         blockcount = self.nodes[0].getblockcount()
         self.stop_nodes()
-        extra_args = [["-reindex-chainstate" if justchainstate else "-reindex"]]
+        extra_args = [["-reindex-chainstate" if justchainstate else "-reindex", "-txindex=0"]]
```

Comment:
should not be `txindex=0` specified only for `-reindex-chainstate`?

```
-extra_args = [["-reindex-chainstate" if justchainstate else "-reindex", "-txindex=0"]]
+extra_args = [["-reindex-chainstate", "-txindex=0"]] if justchainstate else [["-reindex"]]
```

Reply:
IMO keeping it disabled aligns it with test conditions as expected upstream